src/inputs/Switch/Switch.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import { PropsWithChildren, useState } from "react";
+import classNames from "classnames";
+
+import * as styles from "./Switch.module.css";
+
+
+interface SwitchProps {
+  checked?: boolean;
+  disabled?: boolean;
+  onChange(checked: boolean): void;
+  id: string;
+}
+
+export const Switch = ({
+                         disabled = false,
+                         id = "id",
+                         ...props
+                       }: PropsWithChildren<SwitchProps>) => {
+  const [checked, setChecked] = useState(false);
+
+  const getIfDisabledLabel = (): string => {
+    return disabled ? styles.switchLabelDisabled : "";
+  };
+
+  const getIfDisabledSlider = (): string => {
+    return disabled ? styles.switchSliderDisabled : "";
+  };
+
+  const labelClassName = classNames(styles.switchLabel, getIfDisabledLabel())
```

Comment:
**nitpick:** As we started using classNames we definitely can get rid of our methods like `getIfDisabledSlider`, and just do something like:

```
 const labelClassName = classNames({
  switchLabel: true,
  switchLabelDisabled: disabled
})
```